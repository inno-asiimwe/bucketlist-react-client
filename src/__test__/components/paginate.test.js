import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Paginate from '../../components/paginate';

describe('Paginate component', () => {
  it('Matches its snapshor', () => {
    const output = shallow(<Paginate />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});

