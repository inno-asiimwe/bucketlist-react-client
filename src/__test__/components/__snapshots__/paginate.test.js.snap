// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paginate component Matches its snapshor 1`] = `
<div>
  <ul
    className="pagination"
  >
    <li
      className="page-item"
    >
      <button
        className="page-link"
        onClick={[Function]}
      >
        «previous
      </button>
    </li>
    <li
      className="page-item"
    >
      <p
        className="page-link"
      >
        Page undefined of undefined
      </p>
    </li>
    <li
      className="page-item"
    >
      <button
        className="page-link"
        onClick={[Function]}
      >
        Next»
      </button>
    </li>
  </ul>
</div>
`;
